modules/auxiliary/admin/http/wp_gdpr_compliance_privilege_escalation.rb

```diff
@@ -0,0 +1,94 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HTTP::Wordpress
+
+  def initialize(info = {})
+    super(update_info(
+      info,
+      'Name'            => 'WordPress WP GDPR Compliance Plugin Privilege Escalation',
+      'Description'     => %q{
+        The Wordpress plugin GDPR Compliance allows unauthenticated users to execute any
+        action and update any database value.
+        This comes from a lack of validation in the plugin handles in WordPress’s admin-ajax.php
+        functionality, which leads to unauthorized users being abler to trigger these handlers and
+        from a failure to do capability checks when executing its internal action 'save_setting to
+        make configuration changes.
```

Comment:
This should be cut a little shorter, and add in what it actually does.
Something like (spitballin here)
```
        The Wordpress plugin GDPR Compliance allows unauthenticated users to execute any
        action due to a lack of validation against WordPress's admin-ajax.php allowing 
        arbitrary users the ability to make configuration changes.
       This module changes the admin's email address, enables user registration, then
       sets the default user registration level to administrator.
```